cmake/utils/ifm3d-dpkg-deps.py.in

```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- python -*-
 
 #
-# Copyright (C)  2017 Love Park Robotics, LLC
+# Copyright (C)  2017 ifm electronic, gmbh
```

Comment:
Maybe updating the year is also a good idea

Reply:
I am no lawyer, but my understanding is the copyright date is the year of first publication. It also makes it obvious that ifm claims copyright back to 2017 -- if it were updated this date it becomes less clear.

I think you've dealt with this topic more than I have -- is my understanding wrong?

Reply:
Better do not ask a German about Copyright laws https://en.wikipedia.org/wiki/Copyright_law_of_Germany#Transfer I think the right  way to do this would be adding another line
```
# Copyright (C)  2020 ifm electronic, gmbh
# Copyright (C)  2017 Love Park Robotics, LLC
```

Reply:
Ah, I didn't realize that was a think. Definitely like that strategy as it preserves the history of the code better!

Will have to scrub the rest of the codebase and update copyright accordingly

Reply:
I'll add the line on my next push but I've also created a [new issue](https://github.com/ifm/ifm3d/issues/216) for SPDX and the rest of the source so we don't overload this PR